luxonis_ml/data/augmentations/albumentations_engine.py

```diff
@@ -249,6 +249,7 @@ def __init__(
         keep_aspect_ratio: bool = True,
         is_validation_pipeline: bool = False,
         min_bbox_visibility: float = 0.0,
+        seed: int | None = None,
```

Comment:
```suggestion
        seed: Optional[int] = None,
```
because of python3.8